python/orca/src/bigdl/orca/learn/pytorch/pytorch_ray_estimator.py

```diff
@@ -256,10 +257,28 @@ def transform_func(worker, partition_refs):
 
             worker_stats = ray_xshards.reduce_partitions_for_actors(self.remote_workers,
                                                                     transform_func)
+        elif isinstance(data, Dataset):
+            shards = data.split(n=self.num_workers)
```

Comment:
We could also keep the locality for workers.

Reply:
keep the locality? Like output rank0 and rank1 results?

Reply:
Oh I get your point now haha. like `data.split(n, locality)`?

Reply:
> Oh I get your point now haha. like `data.split(n, locality)`?

That's right! 😄